PlaygroundService/Controllers/PlaygroundController.cs

```diff
@@ -153,6 +153,7 @@ public async Task<IActionResult> BuildService(IFormFile contractFiles)
                 var res = Content(Convert.ToBase64String(Read(pathToDll)));
                 
                 _logger.LogInformation("PlaygroundController - BuildProject method over: " + " time:" + DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+                await codeGeneratorGrain.DelData(zipPath, extractPath);
```

Comment:
On a side note, is it better to create a directory based on the GUID of the grain to store all the zip, extracted data and dlls. Then at the end, we simply remove that whole directory instead of deleting zipPath and extractPath seperately? This might be a cleaner implementation. What do you think?